web/src/components/tasks/business-formation/BusinessFormationTextBox.tsx

```diff
@@ -45,7 +45,7 @@ export const BusinessFormationTextBox = (props: Props): ReactElement => {
   return (
     <>
       <div className="flex flex-column mobile-lg:flex-row mobile-lg:flex-align-center margin-bottom-2">
-        <h2 className="h3-styling margin-bottom-0">
+        <h2 className="h3-styling margin-0-override">
```

Comment:
[dust] We do have access to a `margin-0` class. Any reason we opted for the override here?

Reason I'm asking is there are a lot of places it looks like we may be excessively using `important` in our CSS. `margin-0-override` seems to be one of them. I do see both used in the code.